docs/paper/dev/api/entity-api/entity-teleport.mdx

```diff
@@ -3,7 +3,7 @@ slug: /dev/entity-teleport
 description: The entity teleportation API and how to use it.
 ---
 
-# Entity Teleportation
+# Teleportation
 
 Entities can be instantaneously teleported to specific positions, synchronously and asynchronously with the
```

Comment:
first time seeing this page and idk if in scope, but we really should explain when and why to use the async methods (i.e. no sync chunk loads, performance 📈)

Reply:
that's out of scope for this PR, but I could add that in a later one